src/transactions/TransactionEngineTest.java

```diff
@@ -55,5 +59,50 @@ public void testUniquenessAndDigits() {
             }
         }
     }
+    //Now I will test the createPersonMap() methods in the DataManager class by creating a csv file
+        //with a few Person objects in it then create a DataManager class and make sure that a Person map is created
+        //for Person objects that have null fields (such as address) and that a transaction ID is created for the Person
+        //objects in the csv file
+    @Test
+    public void testIDCreatedWithGetPersonMap() {
+        DataManager dataManager = new DataManager("Test-Customers.csv");
+        //now do the createPersonMap() method to create a map of Person objects with their account number
+            //as their key
+        Map<String, Person> personMap = dataManager.createPersonMap();
+        //Now I will get a few of the Person objects in the csv file and store them here to see if there are
+            //transaction IDs for them
+        Person person1 = personMap.get("abc123");
+        Person person2 = personMap.get("success789");
+        Person person3 = personMap.get("16554846");
+        //person1 has several empty fields in the CSV file so we want to make sure a transaction ID is created
+        assertNotNull(person1.getTransactionId());
+        //person2 also has an empty field in the CSV file
+        assertNotNull(person2.getTransactionId());
+        //person3 has no empty fields in the CSV file
+        assertNotNull(person3.getTransactionId());
+    }
+    //Now I will test the getAllCustomerTransactionIDs and getCustomerTransactionID methods
+    //To do this I created a csv file with a few Person objects stored in it and am using this csv file to create
+        //a DataManager object to test these methods
+    @Test
+    public void testGetAllCustomerTransactionIDs() {
+        TransactionEngine transactions = new TransactionEngine("Test-Customers.csv");
```

Comment:
I would recommend hard-coding the CSV file somewhere in the same test, and then writing it to the file system & passing its path here. Also try passing in the CSV in its string format directly to the data manager somehow. If we can’t do this, then we should rethink our design for the `DataManager` class

Reply:
Okay, I can try that tomorrow

Reply:
Actually, don't worry about it